services/node_stake_update.proto

```diff
@@ -37,10 +37,50 @@ message NodeStakeUpdateTransactionBody {
    */
   Timestamp end_of_staking_period = 1;
 
+  /**
+   * The maximum reward rate, in tinybars per whole hbar, that any account can receive in a day.
+   */
+  int64 max_daily_reward_rate = 3;
```

Comment:
Will there be a HIP update to add this property? I don't think it's there currently.

Reply:
That could be done. 👀